package.json

```diff
@@ -68,7 +68,7 @@
     "node": ">=16"
   },
   "dependencies": {
-    "@datadog/native-appsec": "^4.0.0",
+    "@datadog/native-appsec": "4.1.0",
```

Comment:
reminder to change this to 5.0.0 later on

Reply:
why is this resolved, if it is not changed yet?

Reply:
It is, he changed it already